File: теория алгоритмов/if4.cpp
//ƒаны три целых числа. Ќайти количество положительных чисел в исходном наборе.
#include <iostream>
#include <time.h>
#include <stdlib.h>

using namespace std;

int main()
{
int a, b, c, flag;
cin >> a >> b >> c;
if (a > 0){
flag +=1;
}
if (b > 0){
flag +=1;
}
if (c > 0){
flag +=1;
}	
cout << flag;



return 0; // возвращаем 0
}

File: теория алгоритмов/python/if 4.py
#If4. Даны три целых числа.
#Найти количество положительных чисел в исходном наборе.
a = int(input())
b = int(input())
c = int(input())
flag = 0 
for i in a, b, c:
    if i > 0:
        flag += 1
print(flag)
